examples/DataApi-Water-And-Energy-Usage/template/modules/shared_data/src/main/scala/com/my/currency/shared_data/Utils.scala

```diff
@@ -1,36 +1,28 @@
 package com.my.currency.shared_data
 
-import com.my.currency.shared_data.Types.{ UsageState, UsageUpdate}
-import io.circe.parser
-import io.circe.syntax.EncoderOps
+import cats.data.NonEmptySet
+import cats.effect.Async
+import cats.syntax.foldable.toFoldableOps
+import cats.syntax.traverse.toTraverseOps
+import com.my.currency.shared_data.serializers.Serializers
+import com.my.currency.shared_data.types.Types.UsageUpdate
+import org.tessellation.schema.address.Address
+import org.tessellation.security.SecurityProvider
 import org.tessellation.security.hash.Hash
-
-import java.nio.charset.StandardCharsets
+import org.tessellation.security.signature.signature.SignatureProof
 
 object Utils {
 
-  def getUsageUpdateHash(update: UsageUpdate): String = Hash.fromBytes(customUpdateSerialization(update)).toString
-  def customUpdateSerialization(update: UsageUpdate): Array[Byte] = {
-    println("Serialize UPDATE event received")
-    println(update.asJson.deepDropNullValues.noSpaces)
-    update.asJson.deepDropNullValues.noSpaces.getBytes(StandardCharsets.UTF_8)
-  }
-
-  def customStateSerialization(state: UsageState): Array[Byte] = {
-    println("Serialize STATE event received")
-    println(state.asJson.deepDropNullValues.noSpaces)
-    state.asJson.deepDropNullValues.noSpaces.getBytes(StandardCharsets.UTF_8)
-  }
-
-  def customStateDeserialization(bytes: Array[Byte]): Either[Throwable, UsageState] = {
-    parser.parse(new String(bytes, StandardCharsets.UTF_8)).flatMap { json =>
-      json.as[UsageState]
-    }
-  }
+  def getUsageUpdateHash(
+    update: UsageUpdate
+  ): String =
+    Hash.fromBytes(Serializers.serializeUpdate(update)).toString
```

Comment:
I don't think this is correct. Are you sure you don't want this?
```suggestion
    Hash.fromBytes(Serializers.serializeUpdate(update)).value
```

Reply:
I think that they might return the same because it was working. Anyway I'll update